### PR TITLE
Make it compatible with React-like libs

### LINKED ISSUE #736: Make it compatible with React-like libs
Latest versions of react-dropzone introduce the use of event.isDefaultPrevented() which is a jquery-specific[1] function and should be replaced with event.defaultPrevented[2].

[1] https://api.jquery.com/event.isDefaultPrevented/
[2] https://developer.mozilla.org/en-US/docs/Web/API/Event/defaultPrevented

### PR DESCRIPTION
**Pull request by jschill**
Fixes #736

### CODE REVIEW

**Comment by rolandjitsu** [comment]
@jschill there's one test failing.

File: src/utils/index.js [hunk 1]
```diff
@@ -41,6 +41,30 @@ export function allFilesAccepted(files, accept) {
   return files.every(file => fileAccepted(file, accept))
 }
 
+// React's synthetic events has evt.isPropagationStopped,
+// but to remain compatibility with other libs (Preact) fall back
+// to check evt.cancelBubble
+export function isPropagationStopped(evt) {
```
Comment (rolandjitsu) [inline src/utils/index.js]: Please add tests for these 2 functions as well.

**Comment by jschill** [comment]
@rolandjitsu You're right! Sorry about the delay fixing this.

**Pull request merged**